.github/workflows/python-package.yml

```
@@ -5,9 +5,9 @@ name: Python package
 
 on:
   push:
-    # branches: [ $default-branch ]
-  pull_request:
-    # branches: [ $default-branch ]
+    branches:
+      - master  
+    pull_request:
```

actionlint (1.7.4)

unexpected key "pull_request" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)